diffsims/crystallography/__init__.py

```diff
@@ -27,11 +27,13 @@
     get_hkl,
 )
 from diffsims.crystallography.reciprocal_lattice_vector import ReciprocalLatticeVector
+from diffsims.crystallography._diffracting_vector import DiffractingVector
 
 __all__ = [
     "get_equivalent_hkl",
     "get_highest_hkl",
     "get_hkl",
     "ReciprocalLatticePoint",
     "ReciprocalLatticeVector",
+    "DiffractingVector",
```

Comment:
This should be removed so that it is not included in the public API, which it currently is: https://diffsims--205.org.readthedocs.build/en/205/reference/generated/diffsims.crystallography.html.

Reply:
Done